common/src/main/java/com/wynntils/core/features/overlays/Overlay.java

```diff
@@ -99,10 +99,16 @@ public final List<ConfigHolder> getVisibleConfigOptions() {
     }
 
     /** Returns the config option matching the given name, if it exists */
-    public final Optional<ConfigHolder> getConfigOptionFromString(String name) {
-        return getVisibleConfigOptions().stream()
-                .filter(c -> c.getFieldName().equals(name))
-                .findFirst();
+    public final Optional<ConfigHolder> getConfigOptionFromString(String name, boolean visibleOnly) {
+        if (visibleOnly) {
```

Comment:
Why do you need both cases, and not just the configOptions.stream?

Reply:
When I was coding this, premature optimization.. I thought that let's loop over 6 elements, not 9...
I am not sure, perhaps there is some value? Maybe it is fine to remove that argument. What do you think?

Reply:
Or are you suggesting that I should filter the stream if the boolean is set, and not call the other function?